core/src/apps/solana/spl/spl_tokens.py

```
@@ -0,0 +1,24 @@
+from typing import Iterator
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)